partner_firstname/models/partner.py

```diff
@@ -145,3 +188,10 @@ def _install_partner_firstname(self):
         # Force calculations there
         records._inverse_name()
         _logger.info("%d partners updated installing module.", len(records))
+
+    # Python constraint gives a more explicit error
+    _sql_constraints = [(
+        'check_name',
+        "CHECK( 1=1 )",
```

Comment:
It removes this constraint to define it in python

https://github.com/odoo/odoo/blob/9.0/openerp/addons/base/res/res_partner.py#L352